pkgs/applications/misc/far2l/default.nix

```diff
@@ -0,0 +1,58 @@
+{ stdenv, fetchgit, fetchFromGitHub, makeWrapper, cmake, pkgconfig, wxGTK30, glib, pcre, m4, git, bash,
+  xdg_utils, xterm, gvfs, zip, unzip, gzip, bzip2, gnutar, p7zip, xz }:
+
+stdenv.mkDerivation rec {
+  name = "far2l-2.1";
+  rev = "c2f2b89db31b1c3cb9bed53267873f4cd7bc996d";
+
+  src = fetchgit {
+    url = "https://github.com/elfmz/far2l";
+    rev = rev;
+    leaveDotGit = true;
```

Comment:
Do you really need the .git directory? We prefer to use fetchFromGitHub instead of fetchgit.